test/suites/integration/bsp/debt-collection.test.ts

```diff
@@ -257,7 +257,9 @@ describeBspNet(
         "test/cloud.jpg",
         "bucket-1",
         null,
-        null
+        null,
+        null,
+        3 // There are 3 running BSPs to fulfill the storage request
```

Comment:
will accept this only because I am Canadian

<img width="853" alt="image" src="https://github.com/user-attachments/assets/e23bf026-cc3c-4fd4-b3bd-704a704bd81f" />